storm-core/test/clj/backtype/storm/security/auth/nimbus_auth_test.clj

```diff
@@ -48,6 +48,7 @@
         nimbus-server (ThriftServer. (:daemon-conf cluster-map)
                                      (Nimbus$Processor. (:nimbus cluster-map)) 
                                      ThriftConnectionType/NIMBUS)]
+    (Thread/sleep 2000)
```

Comment:
I don't think this was intended change. Removed and ran the test 3 times and it never failed.